components/vf-back-to-top/vf-back-to-top.js

```diff
@@ -0,0 +1,23 @@
+// vf-back-to-top
+
+/**
+ * JS scroll to top functionality
+ * @example vfBack-To-Top(firstPassedVar)
+ * @param {string} [firstPassedVar]  - An option to be passed
+ */
+export function vfBackToTop() {
+
+  const links = document.querySelectorAll(".vf-back-to-top");
```

Comment:
Instead of looking for a class, we try to use JS data selectors, like:

https://github.com/visual-framework/vf-core/blob/f2d1077f48963a1cd7b04248868a185f5294ac53/components/vf-tabs/vf-tabs.js#L13

More on that: stable.visual-framework.dev//guidance/javascript/